test/controllers/ControllerBaseSpec.scala

```diff
@@ -34,5 +34,16 @@ trait ControllerBaseSpec extends MockAuth with Injecting {
       }
     }
   }
+
+  def insolvencyCheck(controllerAction: Action[AnyContent]): Unit = {
```

Comment:
You've added this reusable test function here into the base spec which is good, but you need to call it in each controller spec for each action that goes through the auth predicate.